apps/linkedin-to-notion/example.env

```
@@ -1,5 +1,10 @@
+# Frontend
+PLASMO_PUBLIC_CRX_ID=
+PLASMO_PUBLIC_CRX_PREFIX=chrome-extension://
+
 PLASMO_PUBLIC_SUPABASE_URL=
 PLASMO_PUBLIC_SUPABASE_KEY=
 
```

```suggestion
PLASMO_PUBLIC_NOTION_CLIENT_ID=
```

Used in the content script responsible for adding the nudge during Notion's OAuth workflow ;)

---

It'll be added in this PR then, no need to add it here?

---

This PR has already been merged (my fault). Should I reopen it to make that change?